src/MDMAnimationTraits.h

```diff
@@ -103,22 +103,22 @@
 /**
  The amount of time, in seconds, before this animation's value interpolation should begin.
  */
-@property(nonatomic, assign, readonly) NSTimeInterval delay;
+@property(nonatomic, assign) NSTimeInterval delay;
 
 /**
  The amount of time, in seconds, over which this animation should interpolate between its values.
  */
-@property(nonatomic, assign, readonly) NSTimeInterval duration;
+@property(nonatomic, assign) NSTimeInterval duration;
 
 /**
  The velocity and acceleration of the animation over time.
  */
-@property(nonatomic, strong, nullable, readonly) id<MDMTimingCurve> timingCurve;
+@property(nonatomic, strong, nullable) id<MDMTimingCurve> timingCurve;
```

Comment:
Can you describe the behavior/value if `nil`?

Reply:
Done.